### DISCUSSION TITLE
集群版本从1.24.8升级到1.25.11之后，用sealos add --nodes x.x.x.x 添加节点会报错"sha256 sum not match"

### DISCUSSION THREAD

**Discussion by gitligl** [labring/sealos#3489] [General] [answered]
error Applied to cluster error: failed to copy rootfs default-delzg3yn: failed to copy entry /var/lib/containers/storage/overlay/6ec18d0633c5f6e97bc211bea38fa8140fceea6b5281f52ab8ac3d67e7da53e8/merged/Kubefile -> /var/lib/sealos/data/default/rootfs/Kubefile to 10.20.1.115:22: sha256 sum not match /var/lib/containers/storage/overlay/6ec18d0633c5f6e97bc211bea38fa8140fceea6b5281f52ab8ac3d67e7da53e8/merged/Kubefile(27dabe6a5fdfd470106a75b422f9c6d365ad7ff8a210c94f30c5d8f11e355326) != /var/lib/sealos/data/default/rootfs/Kubefile(0fdf0c11395af7942bc74ae11f606fce11c01c19fa9b19d02d352066ec1cd883), maybe network corruption?

**Comment by gitligl** [comment]
而且/var/lib/containers/storage/overlay/xxxx/merged/Kubefile文件看上去是随机生成的，这个文件是从哪里生成过来的呢？

**Comment by fengxsong** [comment, marked as answer]
set a environment variable first

```
export SEALOS_SCP_CHECKSUM=false
```

then retry
